src/devhub/entity/post/History.jsx

```diff
@@ -7,22 +7,7 @@ props.timestamp: number;
 props.referral: any;
 */
 
-function href(widgetName, linkProps) {
```

Comment:
With this replacement, did you double check that the function call is adjusted? This href function is slightly different from the VM.require href

Reply:
I didn't since the code was from common.jsx, I expected it to be similar to href of VM
I will take a look

Reply:
Fixed it, it works now